File: src/main/java/com/m2t/hachikodictionary/dto/word/LearnedWordDto.java
package com.m2t.hachikodictionary.dto.word;

import com.m2t.hachikodictionary.model.Account;
import com.m2t.hachikodictionary.model.Word;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


@Data
@AllArgsConstructor
@NoArgsConstructor
public class LearnedWordDto {
    private Account account;
    private Word word;
    private int level;
}

File: src/main/java/com/m2t/hachikodictionary/dto/word/LearnedWordDto.kt
package com.m2t.hachikodictionary.dto.word

import com.m2t.hachikodictionary.model.Account
import com.m2t.hachikodictionary.model.Word

data class LearnedWordDto(
    val account: Account,
    val word: Word,
    val level: Int,
) {
}